server/README.md

```diff
@@ -0,0 +1,31 @@
+# mahabharath server
+Documentation: https://documenter.getpostman.com/view/13233153/UUy39mFy#e8ad726b-98de-47a0-982e-f6b9264892e9
```

Comment:
The endpoints for all login, signup, and getting the character details and tokens and all other stuff has to be modified and put by me in netlify as environment variables. 
Am I right?
Just asking because I am just shocked by the PR that you have made.
I know nothing about Docker, yaml, etc etc.. I am new to the web dev.
Please help

Reply:
Yes, I deployed the application on Heroku to maybe guide you through the process, when you deploy it yourself the Base URL will be different.
Sorry for the size of the PR, I included some tools that can help with the development, like docker because I think they can make your life and those who will contribute on the future a lot simpler, they can just start a Docker container and don't stress about setting up the whole environment.

Reply:
that's cool, thanks man!